src/perf/simulations/RawlsSimulation.scala

```diff
@@ -0,0 +1,19 @@
+package default
+
+import io.gatling.core.Predef._
+
+trait RawlsSimulation extends Simulation {
+
+  //function to help us generate TSVs per-run
+  def fileGenerator(f: java.io.File)(op: java.io.PrintWriter => Unit) {
```

Comment:
i really like the idea of tsv outputs from the runs, that will help us to chain together more complex actions